README.rst

```diff
@@ -352,6 +352,14 @@ Optional parameters:
     * ``important`` - mark notifications as important
     * ``silent`` - mark notifications as silent
 
+`Rocket.Chat <https://rocket.chat>`_ - ``rocketchat``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Required parameters:
+    * ``url`` - URL of your Rocket.Chat instance
+    * ``username`` - login username
+    * ``password`` - login password
+    * ``room`` - room/channel name to post in
+
```

Comment:
Add a note here about having to install as `pip install ntfy[rocketchat]` and add it to the extras list above (see Slack for an example)

Reply:
All done. Thanks.